oktaawscli/okta_auth_mfa_app.py

```diff
@@ -99,7 +99,7 @@ def _choose_factor(self, factors):
             # print out the factors and let the user select
             for i, factor in enumerate(factors):
                 factor_name = self._build_factor_name(factor)
-                if factor_name is not "":
+                if not factor_name:
```

Comment:
The warning on the identity check makes sense. 

However, this line is checking that `factor_name` is not empty, so this change is causing the factors to not get printed out:

```
Multi-factor Authentication required for application.
Pick a factor:
Selection:
```

So I'd do this instead:
```suggestion
                if factor_name:
```

Which should restore the factor list, while still using an equality check.
```
Multi-factor Authentication required for application.
Pick a factor:
[ 0 ] token:software:totp( GOOGLE ) : 
[ 1 ] Unknown MFA type: u2f
[ 2 ] token:software:totp( OKTA ) : 
Selection:
```

Reply:
Ah yeah, that makes sense. Updated!

Reply:
Awesome! Thanks!